src/test/java/com/bakdata/kafka/CliTest.java

```diff
@@ -286,7 +286,7 @@ public void run() {
                 .containsEntry("role1", List.of("input3"))
                 .containsEntry("role2", List.of("input4", "input5"));
         assertThat(app.getInputTopics("role1")).isEqualTo(List.of("input3"));
-        assertThat(app.getInputTopic("role2")).isEqualTo("input4");
+        assertThat(app.getInputTopics("role2")).contains("input4");
```

Comment:
Please focus on the changes of the PR and don't fix stuff all over the place. Feel free to submit a clean up PR